sdk/keyvault/Azure.Security.KeyVault.Secrets/src/SecretClient.cs

```diff
@@ -245,11 +245,11 @@ public virtual async Task<Response<SecretBase>> RestoreAsync(byte[] backup, Canc
             return await SendRequestAsync(HttpPipelineMethod.Post, new VaultBackup { Value = backup }, () => new SecretBase(), cancellationToken, SecretsPath, "/restore");
         }
 
-        public virtual Response<Secret> Restore(byte[] backup, CancellationToken cancellationToken = default)
+        public virtual Response<SecretBase> Restore(byte[] backup, CancellationToken cancellationToken = default)
```

Comment:
I see you are applying some of the changes that @schaabs  has in PR https://github.com/Azure/azure-sdk-for-net/pull/6436/files but not all and that might be why you are adding `/` and stuff.
Is the idea to merge this one and then update the other PR? or are you going to take all of the changes in that PR here?

Reply:
I will get the changes from both into this PR :).